src/azul/service/manifest_controller.py

```diff
@@ -137,28 +137,35 @@ def get_manifest_async(self,
 
         if manifest is None:
             location = furl(self_url, args={'token': token.encode()})
+            command_lines = self.service.command_lines(manifest, str(location))
             body = {
                 'Status': 301,
                 'Location': location.url,
                 'Retry-After': token.wait_time
             }
         else:
-            if fetch:
+            try:
+                query_params['objectKey']
+            except KeyError:
                 url = self.manifest_url_func(fetch=False,
                                              catalog=manifest.catalog,
                                              format_=manifest.format_,
                                              filters=json.dumps(manifest.filters),
                                              objectKey=manifest.object_key)
             else:
                 url = manifest.location
+            command_lines = self.service.command_lines(manifest, url)
             body = {
                 'Status': 302,
                 'Location': url,
-                'CommandLine': self.service.command_lines(manifest, url)
+                'CommandLine': command_lines
```

Comment:
Do not like that `command_lines` flows out of this block, both by itself and as part of `body`.

That redundancy is a bit of a smell.

Reply:
Note to self (PL resolution): Use `body['CommandLine']` instead.